examples/kubernetes_orchestration/run.py

```diff
@@ -18,20 +18,18 @@
 from zenml.integrations.facets.visualizers.facet_statistics_visualizer import (
     FacetStatisticsVisualizer,
 )
-from zenml.repository import Repository
 
 if __name__ == "__main__":
-    kubernetes_example_pipeline(
+    pipeline_instance = kubernetes_example_pipeline(
         importer=importer(),
         trainer=svc_trainer(),
         evaluator=evaluator(),
         skew_comparison=skew_comparison(),
-    ).run()
+    )
+    pipeline_instance.run()
 
-    repo = Repository()
-    runs = repo.get_pipeline(pipeline_name="kubernetes_example_pipeline").runs
-    last_run = runs[-1]
-    train_test_skew_step = last_run.get_step(name="skew_comparison")
+    last_run = pipeline_instance.get_runs()[-1]
```

Comment:
This doesn't work right, calling get_runs() on the object returned by a `run()` call?

Reply:
It is the instance of the pipeline, not return of run() though, or am I missing something here?

Reply:
Unless my github UI is messed up, the import to the repo was removed in this file and it is calling the pipeline.run() return object

Reply:
```
pipeline_instance = kubernetes_example_pipeline(
        importer=importer(),
        trainer=svc_trainer(),
        evaluator=evaluator(),
        skew_comparison=skew_comparison(),
    )
    pipeline_instance.run()

    last_run = pipeline_instance.get_runs()[-1]
```
This is how it looks in code, should be fine no?

Reply:
https://github.com/zenml-io/zenml/blob/feature/ENG-1086-adjust-examples-to-new-post-execution-workflow/examples/kubernetes_orchestration/run.py

Reply:
Ahh no I completely messed up, sorry you're right